posthog/models/dashboard_tile.py

```diff
@@ -22,7 +22,15 @@ class Text(models.Model):
     team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
 
 
+class DashboardTileManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().exclude(deleted=True).exclude(dashboard__deleted=True)
+
+
 class DashboardTile(models.Model):
+    objects = DashboardTileManager()
+    including_soft_deleted = models.Manager()
```

Comment:
Hmm, definitely not typical to have two managers on a model. Lots of redundancy. Can we maybe instead add a method like `all_even_soft_deleted()` to `DashboardTileManager()`? That'd be plain `super().get_queryset()`

Reply:
Hmmm 🤔 I pretty much lifted this straight out of the docs https://docs.djangoproject.com/en/4.1/topics/db/managers/#modifying-a-manager-s-initial-queryset

How about rename to `objects_including_soft_deleted`? That way as someone starts to type `objects...` intellisense will show them there are two options?

Reply:
Oh fair enough, looks like I skipped out on that part of the docs :D

Reply:
You've not memorised the entire Django docs 🤯 🤣

Reply:
You bet I've used summaries for the boring required reading books in school